src/renderer/app/app.css

```diff
@@ -1,3 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@700&display=swap');
```

Comment:
I think Google offers a way to group these 2 imports into one single import. You just have to choose everything you need on the font page and it builds the URL for you

Reply:
rigth, done